libcloud/dns/drivers/auroradns.py

```diff
@@ -140,6 +272,14 @@ def list_zones(self):
         return zones
 
     def list_records(self, zone):
+        """
```

Comment:
There is no need to add docstring for methods which are the same as the ones already defined in the base class - those docstrings are automatically inherited from the parent class methods.